### ISSUE TITLE
Docker image?

### ISSUE THREAD

**Issue by Red5d** [ansible-semaphore/semaphore#3]
It's great that this can be run via Vagrant, but is it possible to get a way to run it via Docker as well?

**Comment by matejkramny** [comment]
Should be possible via 'castawaylabs/node-docker' repo..I'll try this tonight, and make it work if it doesn't.Thanks,Matej KramnyOn Mon, Feb 23, 2015 at 6:38 pm,  ansible-semaphore/semaphore <reply@reply.github.com> wrote:            It's great that this can be run via Vagrant, but is it possible to get a way to run it via Docker as well?

**Comment by matejkramny** [comment]
Yh to run this in docker:

1. Create a Redis container (eg name `redisio`)
2. Create a Mongodb container (eg name `mongodb`)
3. Clone semaphore, set up `lib/credentials.json` (copy https://github.com/ansible-semaphore/semaphore/blob/master/lib/credentials.example.json). Set the mongo and redis host accordingly.
3. Run Semaphore in https://github.com/castawaylabs/node-docker

Example:
```
docker run -d \
  --name semaphore \
  -p 8080:80 \
  -e "NODE_ENV=production" -e "PORT=80" \
  -v /wherever/semaphore/is:/srv/app \
  --link redisio:redis --link mongodb:mongo \
  castawaylabs/node-docker
```

A `Dockerfile` in this repo would be potentially more helpful and easier to get started with though. PR's welcome :)

**Comment by matejkramny** [comment]
Does this solve your issue, @Red5d?

**Comment by Red5d** [comment]
Yes, thank you!

**Comment by andrewcstewart** [comment]
Just FYI you should be able to handle the entire orchestration with fig.

**Comment by bibby** [comment]
The Docker image doesn't contain `ansible`. The runner depends on this.

**Comment by matejkramny** [comment]
Reopened as the docker image doesn't work properly

**Comment by ashish235** [comment]
I used the docker images to install, but I 'm not getting anything on the GUI except "Loading...". 

Logs from the semaphore container:

app.js:110 Semaphore listening on port 80
app.js:47 Mongodb connection established
config.js:67 Creating Admin user admin@semaphore.local!
::ffff:172.16.100.71 - - [15/Apr/2015:11:32:29 +0000] "GET / HTTP/1.1" 200 890 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36"
::ffff:172.16.100.71 - - [15/Apr/2015:11:32:29 +0000] "GET /vendor/requirejs/require.js HTTP/1.1" 200 890 "http://172.16.131.252/" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36"
  LESS File error : '../vendor/fontawesome/less/font-awesome.less' wasn't found. Tried - /srv/semaphore/public/vendor/fontawesome/less/font-awesome.less,../vendor/fontawesome/less/font-awesome.less
  LESS File : /srv/semaphore/public/css/semaphore.less 13:0
Error: '../vendor/fontawesome/less/font-awesome.less' wasn't found. Tried - /srv/semaphore/public/vendor/fontawesome/less/font-awesome.less,../vendor/fontawesome/less/font-awesome.less
::ffff:172.16.100.71 - - [15/Apr/2015:11:32:29 +0000] "GET /favicon.ico HTTP/1.1" 200 890 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36"
::ffff:172.16.100.71 - - [15/Apr/2015:11:32:29 +0000] "GET /favicon.ico HTTP/1.1" 304 - "-" "Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2272.118 Safari/537.36"


Kindly fix as I was really excited to find this tool which can replace ansible-tower for me.

**Comment by matejkramny** [comment]
This will definitely be fixed.

Seems like a similar issue to #20

**Comment by matejkramny** [comment]
Closed with #20
